src/main/interface/settings.c

```diff
@@ -393,6 +393,25 @@ static const char * const lookupTableVtxLowPowerDisarm[] = {
 };
 #endif
 
+// DMA spec table. Indices are coherent with respective dmaIndentifier_e value.
+#if defined(STM32F3)
+static const char * const lookupTableDmaChannel[] = {
+    "NONE",
+    "DMA1_CH1", "DMA1_CH2", "DMA1_CH3", "DMA1_CH4",
+    "DMA1_CH5", "DMA1_CH6", "DMA1_CH7",
+    "DMA2_CH1", "DMA2_CH2", "DMA2_CH3", "DMA2_CH4",
+    "DMA2_CH5",
+};
+#elif defined(STM32F4) || defined(STM32F7)
+static const char * const lookupTableDmaStream[] = {
+    "NONE",
+    "DMA1_ST0", "DMA1_ST1", "DMA1_ST2", "DMA1_ST3",
+    "DMA1_ST4", "DMA1_ST5", "DMA1_ST6", "DMA1_ST7",
+    "DMA2_ST0", "DMA2_ST1", "DMA2_ST2", "DMA2_ST3",
+    "DMA2_ST4", "DMA2_ST5", "DMA2_ST6", "DMA2_ST7",
+};
+#endif
```

Comment:
Could we not use the DMA tags for this list - and thus not need the lookup? It would need to have its own getting and setting handler (for the values), but no different to byte, integer or even lookup etc :)

Reply:
Are you suggesting a new CLI command or a new parsing mode `MODE_DMA`?